src/app/page.tsx

```diff
@@ -1,20 +1,3 @@
-'use server';
-import { locales } from '@/i18n/i18next-locales';
-import { fallbackLng } from '@/i18n/i18next-settings';
-import dynamic from 'next/dynamic';
-
-interface PageProps {
-  params: {
-    lng: string;
-  };
-}
-
-const App = dynamic(() => import('./ui/app/App'), { ssr: false });
-
-export default async function Page({ params: { lng } }: PageProps) {
-  if (locales.indexOf(lng) < 0) {
-    lng = fallbackLng;
-  }
-
-  return <App starterVariant="default" />;
+export default async function Page() {
+  return <></>;
```

Comment:
Why do we need this empty page?

Reply:
I need to have a "entry" page. But this will not be used as we make use of locales using `next-i18n-router` in middleware.

Reply:
Tried removing it again, but it would leave me with an error when accessing "/". For some reason, this file is not being picked up though and takes the file from the [lng] folder. But apparently, I can not delete this file?
I left it in

Reply:
Let's return null from here if possible, at least, but it would be nice to investigate the issue further and see what can be put inside this page to avoid having an empty page.